### ISSUE TITLE
Recording issues in erofs rom with kernelsu support.

### ISSUE THREAD

**Issue by mcurly** [chenxiaolong/BCR#477]
Hello and thank you for your work.

I have been trying to search for a working solution/workaround but haven't managed to quite get to it.

I am on a custom android 14 ROM on Courbet device (Xiaomi Mi 11 Lite 4G).

It is EvolutionX ROM, and it has EROFS root file system and has support for KernelSU.

The issues range from a distorted recording up to silent or truncated recording.

Can anyone shed some light in here?

Thank you.

**Comment by chenxiaolong** [comment]
Glad you like BCR!

Please try changing the output format to WAV/PCM. If the same thing happens there, then the issue is very likely to be a bug in the ROM (because BCR, in this mode, records byte-for-byte exactly the audio it received from Android).

In general, the issues you describe (distorted audio, etc.) are caused by broken audio drivers. If you're able to, I'd suggest sticking with something as close to the stock ROM as possible. That usually has a better chance of working. If you run into the same issue there though, then you're unfortunately out of luck.

**Comment by mcurly** [comment]
Hello Gunnerson, thank you for having answered my question. I'll try your
suggestions and will post back.

See you soon!

Cheers.

**Comment by mcurly** [comment]
Hi again @chenxiaolong 

I eventually moved onto some other ROM.

Today I am using crDroid for my device (code-named Courbet). It's even on Android 14 version. I am using other licensed software that I had purchased even before using your notable BCR. That licensed software was not working with that ROM with the same results as your BCR. So I am guessing that the error originated from the ROM itself.

Sorry for this late report.

Cheers!
